.github/workflows/dockerhub-released-image.yml

```diff
@@ -45,4 +49,5 @@ jobs:
         env:
           REGISTRY: karmada
           VERSION: ${{ github.ref_name }}
+          COSIGN_EXPERIMENTAL: 1
```

Comment:
What does this env mean? Is there any reference about it?

Reply:
It's mean use keyless signatures work in Cosign .

Doc from cosign:
https://docs.sigstore.dev/cosign/keyless/#keyless-verifying